picoquic/sockloop.c

```diff
@@ -237,9 +240,15 @@ int picoquic_packet_loop(picoquic_quic_t* quic,
     memset(sock_af, 0, sizeof(sock_af));
     memset(sock_ports, 0, sizeof(sock_ports));
 
-    if ((nb_sockets = picoquic_packet_loop_open_sockets(local_port, local_af, s_socket, sock_af, 
-        sock_ports, socket_buffer_size, PICOQUIC_PACKET_LOOP_SOCKETS_MAX)) == 0) {
+    if ((nb_sockets = picoquic_packet_loop_open_sockets(config->multipath_default_ip, local_port, local_af,
+                                                        s_socket, sock_af, sock_ports, socket_buffer_size,
+                                                        PICOQUIC_PACKET_LOOP_SOCKETS_MAX)) == 0) {
         ret = PICOQUIC_ERROR_UNEXPECTED_ERROR;
+    } else if (config->multipath_alternative_ip != NULL) {
+        nb_sockets += picoquic_packet_loop_open_sockets(config->multipath_alternative_ip, local_port, local_af,
+                                                        s_socket+nb_sockets, sock_af+nb_sockets,
+                                                        sock_ports+nb_sockets, socket_buffer_size,
+                                                        PICOQUIC_PACKET_LOOP_SOCKETS_MAX);
     }
```

Comment:
see discussion.

Reply:
Not a real problem, but as a rule it is better to avoid cosmetic changes in a PR...